src/components/Head.astro

```diff
@@ -42,4 +42,12 @@ const { description, section, title } = Astro.props;
   </script>
   <noscript><p><img src="https://analytics.fusionauth.io/matomo.php?idsite=2&amp;rec=1" style="border:0;" alt="" /></p></noscript>
   <!-- End Matomo Code -->
+  <!-- Google tag (gtag.js) -->
```

Comment:
I'm not sure whether it's important, but according to [Google's instructions](https://developers.google.com/tag-platform/gtagjs/install#add_the_google_tag_to_your_website) the script should be placed "immediately after the opening `head` tag"